### ISSUE TITLE
"'propTypes' is already defined"

### ISSUE THREAD

**Issue by agilgur5** [feross/standard#561]
I happen to have a file with two classes in it:

```
class MyClass1 extends Component {
 static propTypes = { ... }
 ...
}

class MyClass2 extends Component {
  static propTypes = { ... }
  ...
}
```

and standard is throwing "'propTypes' is already defined" on the second declaration, even though they're only defined within the scope of the class, not globally, so this _seems_ like a bug to me

Not 100% sure which dependency/rule may be causing this
Edit: seems like this occurs for any file with multiple classes in it that have `static` properties with the same name (like `propTypes`)

**Comment by dcousens** [comment]
Sounds like eslint?

**Comment by agilgur5** [comment]
Seems like it wasn't `eslint` (https://github.com/eslint/eslint/issues/6635), but a regression in `babel-eslint` (https://github.com/babel/babel-eslint/issues/337) that was fixed 2 days ago with `6.1.2`.
